### PR TITLE
[CYB-4;CYB-137] TableAPI Flink Indexer + Parser UI updates

### PR DESCRIPTION
**Pull request by stas-panasiuk**
TableAPI Flink Hive Indexing implementation;
Parser ui allows user to choose from columns defined in indexing schema

### CODE REVIEW

**Comment by vpavlenko-cv** [comment]
UI looks good

**Review by vpavlenko-cv — APPROVED** [review]

File: flink-cyber/cyber-jobs/src/main/resources/conf/templates/index/index.properties.template [hunk 1]
```diff
@@ -2,6 +2,16 @@ hive.catalog=hive
 hive.schema=cyber
 hive.table=events
 
+#flink.writer=tableapi
+
+#Uncomment one of the following lines in case flink.writer=tableapi is specified
+#flink.output-connector=hive
+#flink.output-connector=kafka
+
+#Uncomment following two lines in case flink.writer=tableapi is specified
```
Comment (carolynduby) [inline flink-cyber/cyber-jobs/src/main/resources/conf/templates/index/index.properties.template:11]: Please change the template so the can uncomment the lines without having to adjust the path.  For example:

#flink.tables-init-file=PIPELINE/triage/table-config.json
#flink.mapping-file=PIPELINE/triage/mapping-config.json

File: flink-cyber/cyber-jobs/src/main/resources/conf/templates/index/index.properties.template [hunk 2]
```diff
@@ -2,6 +2,16 @@ hive.catalog=hive
...
+#flink.tables-init-file=/home/systest/cyb-105-test/main/cyb-105/index/conf/table-config.json
+#flink.mapping-file=/home/systest/cyb-105-test/main/cyb-105/index/conf/mapping-config.json
+
```
Comment (carolynduby) [inline flink-cyber/cyber-jobs/src/main/resources/conf/templates/index/index.properties.template:14]: Also please provide example table-config.json and mapping-config.json files.

File: flink-cyber/flink-indexing/flink-indexing-hive/README.md [hunk 3]
```diff
@@ -80,3 +84,192 @@ The streaming writer creates a transaction and keeps adding events to the transa
 hive.timestamp.format default yyyy-MM-dd HH:mm:ss.SSSSSS,yyyy-MM-dd'T'HH:mm:ss.SSS'Z',yyyy-MM-dd'T'HH:mm:ss.SS'Z',yyyy-MM-dd'T'HH:mm:ss.S'Z',yyyy-MM-dd'T'HH:mm:ss'Z'
 
 Note that the fixed fields include the UUID for the message, the timestamp and message field (a basic string describing the message) and fields (a generic map holding all the fields, flattened).
+
+## TableAPI Writer
+
+TableAPI writer is an optimized version that aims to provide both better performance and and ability to have multiple outputs.
```
Comment (carolynduby) [inline flink-cyber/flink-indexing/flink-indexing-hive/README.md]: correct "and and"

File: flink-cyber/cyber-jobs/src/main/resources/scripts/cs-restart-index [hunk 4]
```diff
@@ -57,4 +57,4 @@ fi
 
 cs-cancel-job ${job_name}
 
-cs-start-job -j flink-indexing-hive -p /etc/cybersec/conf/triage.properties -p kafka.properties -p "$generated_index_props" "${ship_configs[@]}"
+cs-start-job -j flink-indexing-hive -y $pipeline/index/conf -p /etc/cybersec/conf/triage.properties -p kafka.properties -p "$generated_index_props" "${ship_configs[@]}"
```
Comment (carolynduby) [inline flink-cyber/cyber-jobs/src/main/resources/scripts/cs-restart-index]: do the table indexing configurations need to be shipped?
It looks like the client reads them at startup?

**Review by carolynduby — CHANGES_REQUESTED** [review]
Please check the scripts and provide an example config.

**Review by carolynduby — APPROVED** [review]
Looks good.

**Pull request merged**